# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cyberjack/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ctapi-cyberjack-2.0.8/ctapi/Makefile.vanilla	2007-06-17 13:20:58.211563160 +0200
+++ ctapi-cyberjack-2.0.8/ctapi/Makefile	2007-06-17 13:21:35.065663356 +0200
@@ -22,7 +22,7 @@
 	$(CC) $(CFLAGS) -o $@ -c $^
 
 libso: $(OBJS) $(LIBS)
-	ld -x --shared -lusb -o libctapi-cyberjack.so $(OBJS)  $(LIBS)
+	$(LD) -x --shared -lusb -o libctapi-cyberjack.so $(OBJS)  $(LIBS)
 
 liba: $(OBJS) $(LIBS)
 	mkdir tmp 
--- ctapi-cyberjack-2.0.8/pcsc/Makefile.vanilla	2007-06-17 13:21:42.782103091 +0200
+++ ctapi-cyberjack-2.0.8/pcsc/Makefile	2007-06-17 13:21:49.490485380 +0200
@@ -20,7 +20,7 @@
 	install -o root -g root libcyberjack_ifd.so $(INSTALL_PREFIX)/lib
 
 libso: $(OBJS)
-	ld -x --shared -lusb -o libcyberjack_ifd.so $(OBJS) 
+	$(LD) -x --shared -lusb -o libcyberjack_ifd.so $(OBJS) 
 
 liba: $(OBJS)
 	ar r libcyberjack_ifd.a $(OBJS) $(CTOBJS)
